<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>boardWithgame</class>
 <widget class="QDialog" name="boardWithgame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="stepBackButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>320</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/img/previous.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>310</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="text">
    <string>Your score:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>320</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/img/reset.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>370</y>
     <width>431</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" rowstretch="100,0,0,0" columnstretch="100,0,0,0">
    <property name="sizeConstraint">
     <enum>QLayout::SetFixedSize</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>3</number>
    </property>
    <item row="0" column="2">
     <widget class="QLineEdit" name="plate3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>105</width>
        <height>105</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>41</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignJustify|Qt::AlignTop</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="plate5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>105</width>
        <height>105</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>41</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignJustify|Qt::AlignTop</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLineEdit" name="plate16">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>105</width>
        <height>105</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>41</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignJustify|Qt::AlignTop</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="plate6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>105</width>
        <height>105</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>41</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignJustify|Qt::AlignTop</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="plate15">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>105</width>
        <height>105</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>41</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignJustify|Qt::AlignTop</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="plate13">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>105</width>
        <height>105</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>41</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignJustify|Qt::AlignTop</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="plate14">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>105</width>
        <height>105</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>41</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignJustify|Qt::AlignTop</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="plate10">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>105</width>
        <height>105</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>41</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignJustify|Qt::AlignTop</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="plate7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>105</width>
        <height>105</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>41</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignJustify|Qt::AlignTop</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="plate2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>105</width>
        <height>105</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>41</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignJustify|Qt::AlignTop</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="plate8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>105</width>
        <height>105</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>41</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignJustify|Qt::AlignTop</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="plate11">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>105</width>
        <height>105</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>41</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignJustify|Qt::AlignTop</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="plate12">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>105</width>
        <height>105</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>41</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignJustify|Qt::AlignTop</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="plate9">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>105</width>
        <height>105</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>41</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignJustify|Qt::AlignTop</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLineEdit" name="plate1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>105</width>
        <height>105</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>41</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignJustify|Qt::AlignTop</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="plate4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>105</width>
        <height>105</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>41</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignJustify|Qt::AlignTop</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>320</y>
     <width>26</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/img/cross.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="liveScore">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>320</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>527</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Speed mode means that you won`t have animations</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>280</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enable speed mode</string>
   </property>
  </widget>
  <zorder>stepBackButton</zorder>
  <zorder>label</zorder>
  <zorder>resetButton</zorder>
  <zorder>gridLayoutWidget</zorder>
  <zorder>exitButton</zorder>
  <zorder>label_2</zorder>
  <zorder>checkBox</zorder>
  <zorder>liveScore</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
